src/__test__/Details.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { BrowserRouter as Router } from 'react-router-dom';
+import store from '../redux/store';
+import Details from '../components/Details/Details';
+
+describe('CurrencyDetail Component', () => {
+  test('renders correctly when currency details are available', () => {
+    const tree = render(
+      <Provider store={store}>
+        <Router>
+          <Details />
+        </Router>
+      </Provider>,
+    );
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] You have done an amazing job on this milestone but there are just a few requirements you haven't met yet
   - Kindly note that you are also supposed to test some of the pure functions on your redux code. Currently, you have only tested the components.